sansred/sansdata.py

```diff
@@ -34,7 +35,7 @@ def _s(b):
         return b
 
 class RawSANSData(RawVSANSData):
-    suffix = ".sans"
+    suffix = "sans"
```

Comment:
I suggest leaving the dot on the suffix so that the instrument can define an empty suffix if it wants.

Reply:
sure